doc/tutorial_ti_cc3200.md

```diff
@@ -1,283 +0,0 @@
-# How to connect your CC3200 to Xively
```

Comment:
This needs a document in github that redirect/points to the corresponding documement on Readme.io.  Please see the story's acceptance criteria.